core/chains/evm/txmgr/orm.go

```diff
@@ -64,16 +71,30 @@ type EthReceiptsPlus struct {
 }
 
 type orm struct {
-	q      pg.Q
-	logger logger.Logger
+	q         pg.Q
+	logger    logger.Logger
+	ctx       context.Context
```

Comment:
Is this used?

Reply:
Yeah it's used down on line 84-85 when it's passed into pg.Q. The defer function is used in the Close() method

Reply:
But is this `ctx` field accessed again after construction?